scss/_settings_tables.scss

```diff
@@ -0,0 +1 @@
+$table-layout: fixed !default;
```

Comment:
This requires the user to learn the values where as if we are just switching between two it is much better to give them a switch. So this could be called `$table-layout-fixed: true !default;`. This makes it much easier to document.

Reply:
fixed in #2387